lib/terraforming/resource/iam_policy.rb

```diff
@@ -16,24 +16,26 @@ def initialize(client)
       end
 
       def tf
-        apply_template(@client, "tf/iam_policy")
+        apply_template(@client, 'tf/iam_policy')
       end
 
       def tfstate
         iam_policies.inject({}) do |resources, policy|
           version = iam_policy_version_of(policy)
           attributes = {
-            "id" => policy.arn,
-            "name" => policy.policy_name,
-            "path" => policy.path,
-            "description" => iam_policy_description(policy),
-            "policy" => prettify_policy(version.document, breakline: true, unescape: true),
+            'id' => policy.arn,
+            'name' => policy.policy_name,
+            'path' => policy.path,
+            'description' => iam_policy_description(policy),
+            'policy' => prettify_policy(
+              version.document, breakline: true, unescape: true
+            ),
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)